tests/manage/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -70,7 +71,7 @@ def test_non_ocs_taint_and_tolerations(self):
         Test runs the following steps
         1. Taint ocs nodes with non-ocs taint
         2. Set tolerations on storagecluster, subscription, configmap and ocsinit
-        3. Respin all ocs pods and check if it runs on ocs nodes with tolerations
+        3. Check toleration on all ocs pods and hardcode list of pods
```

Comment:
This could be reframed as "Check toleration on all ocs pods"

Reply:
Done